File: Medium/Python/2425. Bitwise XOR of All Pairings.py
class Solution(object):
    def xorAllNums(self, nums1, nums2):
        if len(nums1) % 2 == 0 and len(nums2) % 2 == 0:
            return 0
        n = 0

        if len(nums2) % 2 == 1:
            for num in nums1:
                n ^= num
        if len(nums1) % 2 == 1:
            for num in nums2:
                n ^= num
        
        return n

File: Medium/JavaScript/2425. Bitwise XOR of All Pairings.js
var xorAllNums = function(nums1, nums2) {
    if(nums1.length % 2 == 0 && nums2.length % 2 == 0)
        return 0;
    let n = 0;


    if(nums2.length % 2 == 1){
        for(let num of nums1)
            n ^= num;
    }
    if(nums1.length % 2 == 1){
        for(let num of nums2)
            n ^= num;
    }
    
    return n;
};
